topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/slides.html

```diff
@@ -0,0 +1,204 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+title: Integrate and query local datasets and distant RDF data with AskOmics using Semantic Web technologies
+questions:
+- What is the Semantic Web and  how can it help to integrate and query data?
+- How can AskOmics help to benefit from Semantic Web technologies without having the pain to write RDF or SPARQL code?
+objectives:
+- Understand the basics of RDF and SPARQL
+- Learn how input data must be structured to be integrated with AskOmics
+- Learn how to connect distant SPARQL endpoints to local data with AskOmics
+key_points:
+- RDF and SPARQL are Semantic Web technologies that come useful for data integration and querying but the technical aspects may deter end-users
+- AskOmics is a web platform for data integration and query using Semantic Web in a user-friendly way
+contributors:
+- xgaia
+- abretaud
+- annesiegel
+- odameron
+---
+
+# How to explore data
+
+---
+## Requirements
+
+Study of biological mechanisms requires to:
+
+- **integrate** multiple data sources (differential expression results, genome annotation, remote protein database)
+- **query** them to answer a biological question (which genes are over-expressed in a condition, and what are the protein coded by these genes)
+
+.image-00[ ![](images/from_data_to_result.png)]
+
+---
+
+# What is the Semantic Web?
+
+---
+
+## Semantic Web
+
+Set of recommendations to **integrate data**, to **integrate domain knowledge** and to **perform query** and reasoning.
+
+- Resource Description Framework (RDF): annotate data
+- RDFS + OWL: represent knowledge (data description)
+- SPARQL Protocol and RDF Query Language (SPARQL): query data
+
+---
+
+## RDF
+
+- RDF is for describing **resources** (the R in RDF)
+  - resources are identified by URIs (`nextprot:P01137`, `taxon:9606`)
+
+- **describing** (D in RDF) a resource is representing it explicitly
+  - its attributes (`nextprot:P01137 :hasSequence "MPPSGLRLLL..."`)
+  - its relations to other entities (`nextprot:P01137 :hasTaxon taxon:9606`)
+  - its descriptions (aka classes) (`nextprot:P01137 :is nextprot:Protein`)
+
+---
+
+## RDF: Set of triples
+
+- RDF is represented by triples (subject, predicate, object)
+  - **Subject**: the resource being described
+  - **Predicate**: the relation (from subject to object)
+  - **Object**: a value of the predicate for the subject
+
+.image-01[ ![](images/rdf_schema_1.png)]
+
+```turtle
+nextprot:P01137 :hasTaxon taxon:9606 .
+nextprot:P01137 :hasSequence "MPPSGLRLLL" .
+```
+
+---
+
+## RDF: triples form a labeled directed graph
+
+
+```turtle
+# Description
+nextprot:P01137 rdf:type nextprot:Protein .
+taxon:9606 rdf:type nextprot:Organism .
+# Data
+nextprot:P01137 :hasTaxon taxon:9606 .
+nextprot:P01137 :hasSequence "MPPSGLRLLL" .
+```
+
+.image-02[ ![](images/rdf_schema_2.png)]
```

Comment:
This image is hard to read for me due to thin lettering and low quality, maybe in a future version of this tutorial you could update it?

Reply:
I have uploaded an image with a better resolution